app/views/pages/api-reference/liquid/filters.liquid

```diff
@@ -382,7 +382,7 @@ You can string together `concat` filters to join more than two arrays:
 
 ## date
 
-Converts a timestamp into another date format. The format for this syntax is the same as `strftime`. The input uses the same format as Ruby’s `[Time.parse](https://apidock.com/ruby/Time/parse/class)`.
+Converts a timestamp into another date format. The format for this syntax is the same as `strftime`. They are standard date formatting for UNIX. Available directives for time and date are listed [here](https://devhints.io/strftime).
```

Comment:
In the sentence "They are standard date formatting for UNIX." what does "They" refer to?